Misc/NEWS.d/next/Library/2018-11-11-04-41-11.bpo-22276.Tt19TW.rst

```diff
@@ -0,0 +1,4 @@
+:meth:`~pathlib.Path.glob` and :meth:`~pathlib.Path.rglob` return only
```

Comment:
```suggestion
:class:`~pathlib.Path` methods :meth:`~pathlib.Path.glob` and :meth:`~pathlib.Path.rglob` return only
```
Otherwise it is not clear that it is related to Path and not to the glob module.